<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlItem</class>
 <widget class="QWidget" name="ControlItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="hLabel">
       <property name="text">
        <string>h</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="hDoubleSpinBox">
       <property name="toolTip">
        <string>色相</string>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lLabel">
       <property name="text">
        <string>l</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="lDoubleSpinBox">
       <property name="toolTip">
        <string>亮度</string>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="sLabel">
       <property name="text">
        <string>s</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="sDoubleSpinBox">
       <property name="toolTip">
        <string>饱和度</string>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="hBaseLabel">
       <property name="text">
        <string>hBase</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="hBaseDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lBaseLabel">
       <property name="text">
        <string>lBase</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="lBaseDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="sBaseLabel">
       <property name="text">
        <string>sBase</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="sBaseDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="select_image_button">
       <property name="text">
        <string>选择图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="do_button">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hBaseDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lBaseDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sBaseDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
